README.md

```diff
@@ -6,9 +6,11 @@ This repository is created to give the project a technology refresh while keepin
 
 ### Prerequisites
 
-* Python 3.7+
-* venv
-* Docker and Docker-compose are required if you want to use docker containers
+* [Python 3.7+](https://www.python.org/downloads/)
+* [venv](https://docs.python.org/3/library/venv.html)
+* [Elasticsearch 6.3](https://www.elastic.co/downloads/past-releases/elasticsearch-6-3-0)
+* [Kibana 6.3](https://www.elastic.co/downloads/past-releases/kibana-6-3-0) (Optional)
```

Comment:
Perhaps you should not list Kibana as a prerequisite, since you declare it optional? I'd be tempted to list is separately